src/test/java/bio/terra/integration/DataRepoFixtures.java

```diff
@@ -449,6 +473,37 @@ public List<String> retrieveUserDatasetRoles(TestConfiguration.User user, UUID d
     return validateResponse(response, "retrieving dataset roles", HttpStatus.OK, null);
   }
 
+  // getting a resource's policies
+  public DataRepoResponse<PolicyResponse> retrievePoliciesRaw(
+      TestConfiguration.User user, UUID resourceId, IamResourceType iamResourceType)
+      throws Exception {
+    String pathPrefix;
+    switch (iamResourceType) {
+      case DATASET:
+        pathPrefix = "/api/repository/v1/datasets/";
+        break;
+      case DATASNAPSHOT:
+        pathPrefix = "/api/repository/v1/snapshots/";
+        break;
+      case SPEND_PROFILE:
+        pathPrefix = "/api/resources/v1/profiles/";
+        break;
+      default:
+        throw new IllegalArgumentException(
+            "No path prefix defined for IamResourceType " + iamResourceType);
+    }
```

Comment:
This looks a good place to use a switch expression
```suggestion
    String pathPrefix =
      switch (iamResourceType) {
        DATASET -> "/api/repository/v1/datasets/";
        DATASNAPSHOT -> "/api/repository/v1/snapshots/";
        SPEND_PROFILE -> "/api/resources/v1/profiles/";
        default -> IllegalArgumentException(
            "No path prefix defined for IamResourceType " + iamResourceType);
    }
```

Reply:
I took this suggestion and made some other changes to this method and related ones, e.g. defining the path as a single formatted string.